### ISSUE TITLE
ConnectionPlugin extend connection globally not working as documented

### ISSUE THREAD

**Issue by marceloverdijk** [graphql-nexus/schema#450]
#### Screenshot

![image](https://user-images.githubusercontent.com/197004/84386951-84ff9880-abf2-11ea-82da-8d7702604e17.png)

#### Description 

I've setup the `ConnectionPlugin` as documented like:

```
export const schema = makeSchema({
    types: allTypes,
    outputs: {
        schema: path.join(__dirname, "../schema.graphql")
    },
    plugins: [
        connectionPlugin({
            extendConnection: {
                totalCount: { type: "Int" },
            },
            includeNodesField: true,
        }),
    ],
});
```

and define the connection as:

```
const continents = [
    { id: "africa", code: "AF", name: "Africa", demonym: "African" },
    { id: "antarctica", code: "AN", name: "Antarctica", demonym: "Antarctican" },
    { id: "asia", code: "AS", name: "Asia", demonym: "Asian" },
    { id: "australia", code: "OC", name: "Australia", demonym: "Australian" },
    { id: "europe", code: "EU", name: "Europe", demonym: "European" },
    { id: "north-america", code: "NA", name: "North America", demonym: "North American" },
    { id: "south-america", code: "SA", name: "South America", demonym: "South American" },
];

export const Continent = objectType({
    name: "Continent",
    description: "Represents a continent.",
    definition(t) {
        t.implements("Node");
        t.string("code", {
            description: "The unique code of the continent."
        });
        t.string("name", {
            description: "The name of the continent."
        });
        t.string("demonym", {
            description: "The demonym of the continent."
        });
    },
});

export const continentConnection = queryField((t) => {
    t.connectionField("continentConnection", {
        type: Continent,
        nodes: () => {
            return continents;
        },
        totalCount: () => {
            return continents.length;
        },
    });
});

```

but it seems not executing `totalCount` as it reports the value is `null`.

When I remove `totalCount` from the query the query is executed and returns the data.

**Comment by marceloverdijk** [comment]
I switched this weekend to using Nexus Framework (but as I expected) I see the same issue.

I've setup a simple test project at https://github.com/marceloverdijk/nexus-prisma-country-api/tree/nexus-schema-450

These [lines](https://github.com/marceloverdijk/nexus-prisma-country-api/blob/nexus-schema-450/api/graphql/continent.ts#L55-L56) seem never executed when requesting the `totalCount` field in a query.

**Comment by marceloverdijk** [comment]
Note when using a per-field configuration it __does__ work.

```
    t.connection('continentConnection', {
      type: 'Continent',
      nodes(parent, args, ctx, info) {
        ctx.log.info('nodes(..) called!!!!!')
        return ctx.db.continent.findMany()
      },

      // totalCount(parent, args, ctx, info) {
      //   ctx.log.info('totalCount(..) called!!!!!')
      //   return ctx.db.continent.count()
      // },

      extendConnection(t) {
        t.int('totalCount', {
          resolve: (source, args, ctx) => {
            return ctx.db.continent.count()
          }
        })
      }
    })
```

**Comment by marceloverdijk** [comment]
PS: I see same behaviour for `extendEdge`

```
settings.change({
  schema: {
    connections: {
      default: {
        extendEdge: {
          totalFoo: {
            type: 'Int',
          }
        },
        includeNodesField: true,
      },
    }
  },
})
```

```
    t.connection('continentConnection', {
      type: 'Continent',
      nodes(parent, args, ctx, info) {
        ctx.log.info('nodes(..) called!!!!!')
        return ctx.db.continent.findMany()
      },
      edgeFields: {
        totalFoo(parent, args, ctx, info) {
          ctx.log.info('totalFoo(..) called!!!!!')
          return ctx.db.continent.count()
        },
      },
```

`totalFoo` just returns `null` in the playground and the `totalFoo` method was not called.

Type safety wise everything seems to be ok.
I mean adding `extendEdge` configuration (without the `totalFoo` method) gives a compile error. After adding the compile error is gone, but is seems just not executed.

**Comment by ccincotti3** [comment]
second this.

**Comment by marceloverdijk** [comment]
Thx @ccincotti3 for confirming this as well; at least I'm not the only one.
Hopefully this will get some attention and can be fixed in the future.

**Comment by Aeners** [comment]
I tried too to set `extendConnection.totalCount` globally, and got the same result as you.
My query always resolves: `totalCount: null`.
Any updates on this ?

**Comment by hyusetiawan** [comment]
I have the same issue as well, sounds like whenever `nodes` is set instead of resolve, the extended functions are not called which will return null automatically

**Comment by Weakky** [comment]
I believe this should be fixed by #569. The global connection field extensions were not executed at all
